dev/squash_migrations.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -eo pipefail
+
+cd "$(dirname "${BASH_SOURCE[0]}")/../migrations"
+
+hash migrate 2>/dev/null || {
+    if [[ $(uname) == "Darwin" ]]; then
+        brew install golang-migrate
+    else
+        echo "You need to install the 'migrate' tool: https://github.com/golang-migrate/migrate/"
+        exit 1
+    fi
+}
+
+if [ -z "$1" ]; then
+    echo "USAGE: $0 <migration>"
```

Comment:
```suggestion
    echo "USAGE: $0 <migration>"
    echo ""
    echo "To use this tool, first find the migration version you want to squash. All"
    echo "migrations prior to this version will be squashed into that version."
    echo ""
    echo "For example, if the latest release of Sourcegraph is v3.9.x then since we must"
    echo "maintain backwards compatability for upgrading from 3.7.x and 3.8.x (two minor"
    echo "versions), you should find the last migration that v3.6.x performed and specify"
    echo "that as the version to squash to."
```

Suggest adding help doc like this for clarity for others in the future^

Also, Looking at the migrations from the v3.6.4 branch: https://github.com/sourcegraph/sourcegraph/tree/v3.6.4/migrations

It seems like the one you've chosen is the 2nd to last migration before 3.7 was cut:

![image](https://user-images.githubusercontent.com/3173176/66662879-7fcef180-ebfe-11e9-88dd-ccd47af0ecd1.png)

so if I'm not wrong 1528395582 would be the right one to do